datafusion-examples/Cargo.toml

```diff
@@ -54,6 +78,6 @@ serde = { version = "1.0.136", features = ["derive"] }
 serde_json = { workspace = true }
 tempfile = { workspace = true }
 tokio = { workspace = true, features = ["rt-multi-thread", "parking_lot"] }
-tonic = "0.11"
```

Comment:
is this change related?

Reply:
I tried to capture this in the issue:
> There is an API incompatibility between tonic 0.10 and 0.11. We have 0.10 as a transitive dependency of arrow-flight, and datafusion-examples pulls in 0.11 directly. As a result the flight examples won't compile.
If that's not clear, I'm happy to clarify further.